array/DNAm/preprocessing/filterToAutosomes.r

```diff
@@ -0,0 +1,110 @@
+##---------------------------------------------------------------------#
+##
+## Title: Remove Sex Specific Probes
+##
+##
+##---------------------------------------------------------------------#
+
+
+#----------------------------------------------------------------------#
+# NOTES
+#----------------------------------------------------------------------#
+# project folder is provided on command line
+# ref directory is provided on command line
+# manifest file for epic data is already available in the gds object
+
+
+#----------------------------------------------------------------------#
+# DEFINE PARAMETERS
+#----------------------------------------------------------------------#
+
+args<-commandArgs(trailingOnly = TRUE)
+dataDir <- args[1]
+refDir <- args[2]
+
+
+gdsFile <-paste0(dataDir, "/2_gds/raw.gds")
+qcOutFolder<-file.path(dataDir, "/2_gds/QCmetrics")
+normgdsFile<-sub("\\.gds", "Norm.gds", gdsFile)
```

Comment:
Why use `sub` when we can just use: `paste0(dataDir, "/2_gs/rawNorm.gds")`?

This way each line would be consistent with one another. The `sub` isn't really adding anything except extra complexity.